db/migrate/20220720190421_add_type_to_notifications.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class AddTypeToNotifications < ActiveRecord::Migration[6.1]
+  def up
+    add_column :notifications, :type, :string
```

Comment:
Should probably be not nullable and have an index if you're doing queries based on this column.

Reply:
9103340c